docs/contributing/feature-flags.md

```diff
@@ -1,8 +1,3 @@
----
```

Comment:
❓ Why open this up to everyone? We do not want to encourage usage of this except internally.

Reply:
The metric I use for internal vs external is whether or not special access is required. The feature flags is in the AGPL licensed section of the codebase and not blocked by any employee-specific secrets.

I don't see why we wouldn't want external contributors feature flagging their contributions, if it makes sense and they are able.

Reply:
They're not able, since we'd have to register flags in LD. If you think an engineer would connect with them to do that then alright, but it's a bit of a jump from today.

There's sometimes an eagerness to adopt features in the community and what's happened is users start trying to put these developer overrides in and have problems, therefore increasing support needs. I want the impetus to be on our internal engineering team to quickly remove these in a follow-up release instead.